packages/mobile/src/identity/commentEncryption.test.ts

```diff
@@ -0,0 +1,259 @@
+import {
+  checkTxsForIdentityMetadata,
+  decryptComment,
+  embedPhoneNumberMetadata,
+  encryptComment,
+  extractPhoneNumberMetadata,
+  getCommentKey,
+} from 'src/identity/commentEncryption'
+import { lookupAttestationIdentifiers } from 'src/identity/contactMapping'
+import { PhoneNumberHashDetails } from 'src/identity/privacy'
+import { e164NumberToAddressSelector, e164NumberToSaltSelector } from 'src/identity/reducer'
+import { privateCommentKeySelector } from 'src/web3/selectors'
+import { getMockStoreData } from 'test/utils'
+import {
+  mockAccount,
+  mockAccount2,
+  mockComment,
+  mockE164Number,
+  mockE164NumberHashWithSalt,
+  mockE164NumberSalt,
+  mockPrivateDEK,
+  mockPrivateDEK2,
+  mockPublicDEK,
+  mockPublicDEK2,
+} from 'test/values'
+
+jest.mock('crypto', () => ({
+  ...jest.requireActual('crypto'),
+  randomBytes: jest.fn(() => Buffer.from(new Uint8Array(16).fill(1))),
+}))
+
+const simpleComment = 'Simple comment'
+const simpleCommentEnc =
+  'BNFXzyIGjZqqNyq6r35aV2HlMMqUbGnIqboReD77MwAlI5IyzqLQ99WF5B1bsZSVS1K+7trtJtKGhIdI1vbSJSsBAQEBAQEBAQEBAQEBAQEBCVYJWqi/TZNXbAR9ziyX5MJCtfdulxA1tjlvHR/xpE6WnlC/kyXAfKIMqgKGJXchBNFXzyIGjZqqNyq6r35aV2HlMMqUbGnIqboReD77MwAlI5IyzqLQ99WF5B1bsZSVS1K+7trtJtKGhIdI1vbSJSsBAQEBAQEBAQEBAQEBAQEBBhjruDecYg9fsrPNcQbI3AkcvWra1MHIeOZlcycn7Vqtx+UVNR59A3kqdIDbLuGiAQEBAQEBAQEBAQEBAQEBAQ13jJ/iheQ05aIkSedfLcnSa3Rtr4dsbNt59TAZsQqGxPClubtekbOKaHz7lE5zyI803qP65frfyyCxuQ=='
+const simpleCommentWithMetadata = 'Simple comment~+14155550000piWqRHHYWtfg9'
+const simpleCommentWithMetadataEnc =
+  'BNFXzyIGjZqqNyq6r35aV2HlMMqUbGnIqboReD77MwAlI5IyzqLQ99WF5B1bsZSVS1K+7trtJtKGhIdI1vbSJSsBAQEBAQEBAQEBAQEBAQEBCVYJWqi/TZNXbAR9ziyX5MJCtfdulxA1tjlvHR/xpE6WnlC/kyXAfKIMqgKGJXchBNFXzyIGjZqqNyq6r35aV2HlMMqUbGnIqboReD77MwAlI5IyzqLQ99WF5B1bsZSVS1K+7trtJtKGhIdI1vbSJSsBAQEBAQEBAQEBAQEBAQEBBhjruDecYg9fsrPNcQbI3AkcvWra1MHIeOZlcycn7Vqtx+UVNR59A3kqdIDbLuGiAQEBAQEBAQEBAQEBAQEBAQ13jJ/iheQ05aIkSedfLcnSa3Rtr4dsbNt59TBTlRnqnaAoz0HyOyScr6ckQczlhtxwBqKgGuXE9OnjyeH4PIJ9yutViWOO+glrLIFjpDyl+7FebiJv/p9Ay21LHpUd7b5lSV4PWPDxamBMp6MHRdA='
+const complexComment = 'long comment with emoji 😇🤠👳🏽‍♂️~ test TEST test'
+const complexCommentEnc =
+  'BNFXzyIGjZqqNyq6r35aV2HlMMqUbGnIqboReD77MwAlI5IyzqLQ99WF5B1bsZSVS1K+7trtJtKGhIdI1vbSJSsBAQEBAQEBAQEBAQEBAQEBCVYJWqi/TZNXbAR9ziyX5MJCtfdulxA1tjlvHR/xpE6WnlC/kyXAfKIMqgKGJXchBNFXzyIGjZqqNyq6r35aV2HlMMqUbGnIqboReD77MwAlI5IyzqLQ99WF5B1bsZSVS1K+7trtJtKGhIdI1vbSJSsBAQEBAQEBAQEBAQEBAQEBBhjruDecYg9fsrPNcQbI3AkcvWra1MHIeOZlcycn7Vqtx+UVNR59A3kqdIDbLuGiAQEBAQEBAQEBAQEBAQEBATJ3ip/hhfM0qaIiSahfI8nQa3xtrId9bJV59jBElUbqxKA8zxXyYyTGr/gkHcz1htGoMXyuwrUZoDHHFawgk1046uFz352n+l1rPoFhpHal6vE9k3lR9wF0GN/aWTe2L2g/hp8QQdlwuCgsZnjOVB7hDjSvOZ68mz7WqNk29RT57sb3nMkV'
+
+const phoneDetails: PhoneNumberHashDetails = {
+  e164Number: mockE164Number,
+  salt: mockE164NumberSalt,
+  phoneHash: 'hash',
+}
+
+describe('Encrypt Comment', () => {
+  it('Empty comment', async () => {
```

Comment:
nit: no big deal but we are inconsistently naming tests in this file.
From what I've seen, we usually try to use a verb after `it` in most other test files.

Reply:
Will fix :)